utils/build/build.sh

```diff
@@ -221,11 +227,10 @@ build() {
                 ${DOCKER_PLATFORM_ARGS} \
                 -f ${DOCKERFILE} \
                 -t system_tests/weblog \
-		--pull \
+                $IMAGE_PULL_SOURCE \
```

Comment:
@smola , you [added](https://github.com/DataDog/system-tests/commit/8b1f42cbd5371fce292d1352112a8d16d76c5289) the `--pull` option for weblog, but if I read your comment, it was mainly for the agent.

We are struggling when we need to update weblog base images. I'd like to simply remove this `--pull` option for the weblog, is it ok?

Reply:
@cbeauchesne Yes. `--pull` was relevant for the agent because people were inadvertently using very old agents to test locally. I think it is ok to remove it for weblog if it's causing any problem.

Reply:
Thanks @smola .

@robertomonteromiguel Then you can remove all the `IMAGE_PULL_SOURCE` stuffm and simply remove the `--pull` for weblog

Reply:
@cbeauchesne Ok. I remove the --pull option and It works for both cases: image from local or image from registry